src/ConnectedMode/Hotspots/ILocalHotspotStoreMonitor.cs

```diff
@@ -0,0 +1,31 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+namespace SonarLint.VisualStudio.ConnectedMode.Hotspots
+{
+    /// <summary>
+    /// Marker interface for the component that listens for changes to the local hotspots
+    /// and triggers showing/hiding the local hotspots tool window
+    /// </summary>
+    public interface ILocalHotspotStoreMonitor
+    {
+        System.Threading.Tasks.Task InitializeAsync();
```

Comment:
The implementation has to be in `IssueViz.Security`because of the types used, but the class is created by the ConnectedMode package, so this interface has to be in the ConnectedMode project.